proposals/0101-halve-rent.md

```diff
@@ -0,0 +1,77 @@
+---
+simd: '0101'
+title: Halve Rent
+authors:
+  - Brooks Prumo (Solana Labs)
+category: Standard
+type: Core
+status: Draft
+created: 2024-01-05
+feature: https://github.com/solana-labs/solana/issues/34656
+---
+
+## Summary
+
+Halve rent.
+
+## Motivation
+
+Projects that build on Solana face operational cost volatility if they
+manage/create accounts for their users.  This is because creating accounts
+incurs *rent*, and that cost is both (1) fixed, and (2) denominated in SOL.
+
+The original cost of rent was based on calculations assuming 1 SOL == 1 USD.
+As the SOL token accrues more value, one side effect is that rent increases
```

Comment:
concept nack.

chasing number is not economics.

a more valuable thing should cost more. did we actually make state cheaper somehow or are we going to see a "double rent" when number go back down?